src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TileJSONBuilder.java

```diff
@@ -39,6 +38,9 @@
 import org.geowebcache.grid.GridSubset;
 import org.geowebcache.layer.TileLayer;
 import org.geowebcache.layer.meta.LayerMetaInformation;
+import org.geowebcache.layer.meta.TileJSON;
+import org.geowebcache.layer.meta.VectorLayerMetadata;
+import org.geowebcache.layer.meta.VectorLayerMetadata.GeometryType;
```

Comment:
This feels like it should be split out - its vaguely related, but its really a different change.